src/api/pricer/CoinBase.ts

```diff
@@ -34,10 +34,8 @@ export default class CoinBase extends BaseExchangeApi {
     const targetCurrencyName = targetCurrency.toUpperCase();
 
     try {
-      const url = new URL(this._priceApiPath, this._url);
-      url.searchParams.append('currency', sourceCurrencyName);
       response = await this._httpWrapper.sendPromise<CoinBaseResponse>(
-        url.toString()
+        `${this._url.toString()}?currency=${sourceCurrencyName}`
```

Comment:
I think that in this case it was right to create a new one, since we adding a new search param there

Reply:
updated